lua/damagelogs/client/listview.lua

```diff
@@ -225,10 +225,12 @@ function Damagelog:AddRoleLine(listview, nick, role)
             if role < 0 then
                 v:SetTextColor(role_colors[DAMAGELOG_ROLE_DISCONNECTED])
             else
-                if not ROLES then
-                    v:SetTextColor(role_colors[role])
-                else
+                if TTT2 then
                     v:SetTextColor(GetRoleByIndex(role).color)
```

Comment:
Do we need to handle the situation where a TTT2 role is removed and `GetRoleByIndex(role)` returns nil?

Reply:
See my response here https://github.com/BadgerCode/tttdamagelogs/pull/57#discussion_r663535446